### PR TITLE
DELF documentation updates

### PR DESCRIPTION
**Pull request by andrefaraujo**
# Description

* Centralizing installation instructions
* Requesting TF2.2+
* Refactoring verbose "Code overview" README section
* Acknowledging recent contributors

## Type of change

For a new feature or function, please create an issue first to discuss it
with us before submitting a pull request.

Note: Please delete options that are not relevant.

- [X] Documentation update
- [X] TensorFlow 2 migration

## Tests

None, only minor/documentation changes.

## Checklist

- [X] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [X] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [X] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [X] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: research/delf/README.md [hunk 1]
```diff
@@ -182,104 +182,55 @@ directories therein, `protos` and `python`.
...
-    proto, used for DELF feature extraction.
--   `detector.py` is a module to construct an object detector function.
--   `extract_boxes.py` enables object detection from a list of images.
--   `extract_features.py` enables DELF extraction from a list of images.
--   `extractor.py` is a module to construct a DELF/DELG local feature extraction
-    function.
--   `match_images.py` supports image matching using DELF features extracted
-    using `extract_features.py`.
+This directory contains files for several different purposes, such as:
+reading/writing tensors/features (`box_io.py`, `datum_io.py`, `feature_io.py`);
```
Comment (dan-anghel) [inline research/delf/README.md:193]: ',' instead of ';'

File: research/delf/README.md [hunk 2]
```diff
@@ -182,104 +182,55 @@ directories therein, `protos` and `python`.
...
+local feature aggregation extraction and similarity computation
+(`feature_aggregation_extractor.py`, `feature_aggregation_similarity.py`) and
+helper functions for image/feature loading/processing (`utils.py`,
+`feature_extractor.py`).
 
-The subdirectory `delf/python/delg` contains sample scripts/configs related to
-the DELG paper:
+The subdirectory `delf/python/examples` contains sample scripts to run DELF/DELG
+feature extraction/matching (`extractor.py`, `extract_features.py`,
+`match_images.py`), and object detection (`detector.py`, `extract_boxes.py`).
```
Comment (dan-anghel) [inline research/delf/README.md:201]: remove ','

File: research/delf/README.md (hunk 1 above)
Comment (andrefaraujo) [reply research/delf/README.md]: Done

File: research/delf/README.md (hunk 2 above)
Comment (andrefaraujo) [reply research/delf/README.md]: Done

**Comment by andrefaraujo** [comment]
Thanks Dan! Just fixed the commas :)

**Pull request merged**